scripts/utils/trading_strategy_helpers.js

```diff
@@ -609,15 +588,13 @@ withdrawal of the desired funds
     buildWithdrawAndTransferFundsToMaster,
     buildBracketTransactionForTransferApproveDeposit,
     buildTransferApproveDepositFromOrders,
-    calculateBuyAndSellAmountsFromPrice,
```

Comment:
Does this no longer exist? I am currently using this function in the `synthetix-liqudity` script.  I suppose it is my own fault for not having a test for this, but I believe there are at least two places in the code base depending on this function so we had better make sure to accommodate for this change. Alternatively, and probably more appropriately considering the size of this change already, I might propose that this be removed along with the necessary adjustments in a separate PR.

Reply:
After digging deeper, I see that there is only one file remaining that still needs to be adjusted for this change. That is here. 

https://github.com/gnosis/dex-liquidity-provision/blob/2112744c69e02754d53fdc02ea4b7dce06ac4cbf/scripts/synthetix/facilitate_trade.js#L62-L64

I will make sure to include a unit test for this script ASAP, but we need to be careful of things like this.

Reply:
This was taken care of now in #136